cmd/injective-price-oracle/oracle.go

```diff
@@ -201,12 +218,61 @@ func oracleCmd(cmd *cli.Cmd) {
 			log.Infof("found %d dynamic feed configs", len(dynamicFeedConfigs))
 		}
 
+		storkFeedConfigs := make([]*oracle.StorkFeedConfig, 0, 10)
+		if len(*storkFeedsDir) > 0 {
+			err := filepath.WalkDir(*storkFeedsDir, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					return err
+				} else if d.IsDir() {
+					return nil
+				} else if filepath.Ext(path) != ".toml" {
+					return nil
+				}
+
+				cfgBody, err := os.ReadFile(path)
+				if err != nil {
+					err = errors.Wrapf(err, "failed to read stork feed config")
+					return err
+				}
+
+				feedCfg, err := oracle.ParseStorkFeedConfig(cfgBody)
+				if err != nil {
+					log.WithError(err).WithFields(log.Fields{
+						"filename": d.Name(),
+					}).Errorln("failed to parse stork feed config")
+					return nil
+				}
+
+				storkFeedConfigs = append(storkFeedConfigs, feedCfg)
+
+				return nil
+			})
+
+			if err != nil {
+				err = errors.Wrapf(err, "stork feeds dir is specified, but failed to read from it: %s", *storkFeedsDir)
+				log.WithError(err).Fatalln("failed to load stork feeds")
+				return
+			}
+
+			log.Infof("found %d stork feed configs", len(storkFeedConfigs))
+		}
+
+		storkWebsocket, err := ConnectWebSocket(ctx, *websocketUrl, *websocketHeader)
+		if err != nil {
+			err = errors.Wrapf(err, "can not connect with stork oracle websocket")
+			log.WithError(err).Fatalln("failed to load stork feeds")
```

Comment:
This should not be a fatal error; if the Stork feed WS is down for whatever reason this would prevent the whole Injective oracle from starting, if we cannot connect it should disable pulling/pushing Stork oracle prices. Ideally, it should try to reconnect so it can eventually resume the operations

Reply:
yes, this should be fine 👍